### DISCUSSION TITLE
Chunks not showing or being stiched togeter

### DISCUSSION THREAD

**Discussion by bouzafr** [aclap-dev/video-downloadhelper#748] [Help and Q&A]
### Your question

Hello,

I noticed w/ this new version websites that have chunks of streams (different URLs? don't know the proper name) are not being stiched together at the end. basically every 6 secs there is a new video and downloader helper only downloads one of 6s.

Any ideas?

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"b0bf2c2bf16d412e"}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "/Users/X/Downloads/tmp",
    "concurrent_downloads_max": 10,
    "show_success_notification": false,
    "show_success_notification_for_icognito": false,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 1
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You're going to have to provide the URL of at least one example page on which this happens.  Don't say a web site.  Provide the URL of a specific page on which the problematic video resides.  If there's lots of them, pick one & post its URL.  I have seen this & in some cases -- I repeat, SOME cases -- I have been able to get the content with ffmpeg.  But I can't try that unless you give me a URL to work with.
